### ISSUE TITLE
forward_transform

### ISSUE THREAD

**Issue by serhannn** [tensorflow/compression#2]
Hi. 
This is probably a very stupid question but where is the function `y = forward_transform(x)` defined? I'm getting an error that it is not defined. Is that supposed to be replaced by some custom-defined forward transform function?
Thanks!

**Comment by jonycgn** [comment]
Hi Serhan,

you are correct. The forward and backward transforms are currently left as a pseudocode placeholder. I'll take this as a suggestion to flesh out the example a bit more. We'll likely do this when we get to expanding the library a bit more (should happen soon-ish).

You can just define any appropriate CNN. As a tip, it generally helps to drop the activation function before the bottleneck, and *not* to use batchnorm!

Best!
